src/com/rcv/CastVoteRecord.java

```diff
@@ -97,14 +100,17 @@ public boolean isExhausted() {
   // function: getFractionalTransferValue
   // purpose: getter for fractionalTransferValue
   // returns: value of field
-  public float getFractionalTransferValue() {
+  public BigDecimal getFractionalTransferValue() {
+    if(fractionalTransferValue == null) {
```

Comment:
In general for stylistic or simple readability improvements, if possible I would prefer that you go ahead and just make those changes in code.  I appreciate the time you put into writing these comments, but combined with the time it takes me to read and respond and it will be quicker.

Reply:
If it's a one-off, perhaps -- although that means I have to write it down somewhere and then wait for you to commit your changes, then remember to go back and update it. You can also reject a suggestion if you think it's unhelpful, of course.

For general things (e.g. how to format multi-line function calls), the ideal outcome from my perspective is that we can agree on style guidelines and produce consistent code... so it feels worth my time to point things out, because I'm hoping that discussion leads to a consensus for the future code that we write.

Reply:
I agree with you on having a conversation, if there's a conversation to have, e.g. ternary statement formatting.  If it's something we've agreed on (e.g. putting a space after ifs) or simple readability changes or cleanup, one can simply commit those to the branch.  That's what I'm asking for.  No need to write them down for later, that would be far worse!

Reply:
OK, sounds good!